jobs/tests/templates/bin/pre-start

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 set -eu
+set -x
```

Comment:
Combine this into `set -eux` if you realllly need to keep it, otherwise lose the `-x`